code/game/mecha/mecha_topic.dm

```diff
@@ -306,6 +307,24 @@
 		maint_access = !maint_access
 		send_byjax(src.occupant,"exosuit.browser","t_maint_access","[maint_access?"Forbid":"Permit"] maintenance protocols")
 
+	if (href_list["toggle_port_connection"])
+		if(internal_tank.connected_port)
+			if(internal_tank.disconnect())
+				occupant_message("Disconnected from the air system port.")
+				log_message("Disconnected from gas port.")
+			else
+				occupant_message("<span class='warning'>Unable to disconnected from the air system port!</span>")
+				return
+		else
+			var/obj/machinery/atmospherics/components/unary/portables_connector/possible_port = locate(/obj/machinery/atmospherics/components/unary/portables_connector) in loc
+			if(internal_tank.connect(possible_port))
+				occupant_message("Connected to the air system port.")
+				log_message("Connected to gas port.")
+			else
+				occupant_message("<span class='warning'>Unable to connected with air system port!</span>")
+				return
+		send_byjax(src.occupant,"exosuit.browser","t_port_connection","[internal_tank.connected_port?"Disconnect from":"Connect to"] gas port")
```

Comment:
remove `src.`

Reply:
Should I remove all this interences from file?